experimental/generation/generator/packages/library/test/forms/unittest_array_enum.schema

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "properties": {
+        "EnumArrayColor": {
+            "type": "array",
+            "items": {
+                "type": "string",
+                "enum": [
+                    "yellow",
+                    "red",
+                    "blue",
+                    "black",
+                    "white"
+                ],
+                "maxItems": 3
```

Comment:
>maxItems [](start = 17, length = 8)

Do we enforce this?

Reply:
Do you mean if the keyword maxItems works ?